cmd/src/main.rs

```diff
@@ -109,6 +109,8 @@ enum Cli {
         package: bool,
         #[structopt(long, help = "Save generated data to file system")]
         persist: bool,
+        #[structopt(long, help = "Use real networking")]
```

Comment:
Since this will show up in the help text, should probably improve this very unhelpful explanation... thinking...